Example/Backpack/BPKAppDelegate.m

```diff
@@ -39,11 +43,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
         BPKThemeContainerController *themeContainerController =
             [[BPKThemeContainerController alloc] initWithThemeDefinition:activeTheme
-                                                  rootViewController:self.window.rootViewController];
+                                                      rootViewController:self.window.rootViewController];
         self.window.rootViewController = themeContainerController;
         [self.window makeKeyAndVisible];
     }
 
+    [MSAppCenter start:@"821e66b7-2bf8-4fd6-9f12-a900b282aaf8" withServices:@ [[MSAnalytics class], [MSCrashes class]]];
```

Comment:
Need to revoke this secret and inject it at build time. Not super critical though as this token basically only allows someone to screw with our app stats)

Reply:
Ok turns out that AppCenter doesn't actually have a way of revoking.
https://github.com/microsoft/appcenter/issues/397

And they don't think it's necessary as it's not really a "secret" as such...